### PR TITLE
docs(java): add installation and integration guide for java-sdk

### PR DESCRIPTION
**Pull request by Sarthak160**
Signed-off-by: Sarthak Shyngle <50234097+Sarthak160@users.noreply.github.com>

# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by nehagup** [comment]
@Sarthak160 can you please signoff your commit? DCO check is failing

**Comment by Sarthak160** [comment]
> @Sarthak160 can you please signoff your commit? DCO check is failing

Done

**Comment by nehagup** [comment]
@Sarthak160 have you tried running `yarn start` after making the changes? Currently, on this branch I'm getting this error which is breaking the website.
Screenshot 2022-08-31 at 12 41 42 AM

File: docs/java/installation.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        3. Make some API calls.
+
+- **Run the testcases**
+    - **Note:** Before running tests stop the sample application.
+        - Using environment variable
+            1. Set `KEPLOY_MODE = test` (default "record")
+            2. Run your application.
+            3. You can also run the application with coverage to see the test coverage.
+        - Using command line -
+            1. Add Jacoco [dependency]() and [plugin]().
```
Comment (nehagup) [inline docs/java/installation.md:57]: Please add links for dependency and plugin.

**Review by nehagup — CHANGES_REQUESTED** [review]
This PR needs work on - 

- Remove WIP and Github Issue link from where it is mentioned across the website. 
- Add the Java directory and subsequent pages to `docusaurus.config.js` and `sidebar.js`. 
- Currently added doc page looks like a readme. It needs formatting according to the documentation website.

File: docs/java/installation.md (hunk 1 above)
Comment (Sarthak160) [reply docs/java/installation.md]: @nehagup  I have made some changes mentioned above , kindly review .

**Review by nehagup — APPROVED** [review]
LGTM! 👏

**Pull request merged**